pkg/events/definitions.go

```diff
@@ -39,6 +39,7 @@ func (d *definition) With(options ...Option) Builder {
 
 var defaultOptions = []Option{
 	WithVisibility(ttnpb.RIGHT_ALL),
+	WithAuthFromContext(),
```

Comment:
If this is supposed to be always there, why is it an option?

Reply:
Now that you mention this, I'm not actually sure it should be included by default. Auth information should only be added on events triggered by external calls (registry CRUD, maybe downlink pushes if we have auth context there). So it would indeed be better to set this option on those events directly.